src/gui/input_mac.cpp

```diff
@@ -48,13 +48,27 @@ QKeyEvent CreatePlatformNormalizedKeyEvent(
 	Qt::KeyboardModifiers mod,
 	const QString& text) noexcept
 {
+	
+	std::vector<QString> removeAltCharList  {"¡","“","¶","¢","[","]","|","{","}","≠","¿","±","‘","–","…","∞","µ","~","∫","√","ç","≈","¥","≤","å","‚","∂","ƒ","©","ª","º","∆","@","œ","æ","‘","±","•","π","⁄","¨","Ω","†","®","€","∑","«","„","\"","¿"};
```

Comment:
```
static const std::vector<QChar> removeAltCharList{  '{', ''}' ... }; /*constexpr is even better, but may not work?*/
...
if (std::find(removeAltCharList.begin(), removeAltCharList.end(), c ) != removeAltCharList.end()) {
```

`QChar` should be considerably faster/smaller than `QString`.

No heap allocation, much smaller memory footprint.

Reply:
done